spec/requests/api/v1/ecf_users_spec.rb

```diff
@@ -73,6 +73,19 @@
         expect(parsed_response["data"][0]["attributes"]["cohort"]).to eql(2021)
       end
 
+      context "there is a user without a cohort" do
+        before do
+          school = create(:school)
+          teacher = create(:user, :teacher)
+          create(:participant_profile, participant_type: :ect, school: school, teacher_profile: teacher.teacher_profile)
```

Comment:
We no longer set `school` or `cohort` on ParticipantProfile::ECT, but rather `school_cohort` which captures both

Reply:
Hmm, struggling to think of a test to add instead of this now, this is how you would set up a user to break the previous implementation but now it is impossible to create a profile without a school cohort, or a school cohort without a cohort 🤔